earthpy/tests/test_plot.py

```diff
@@ -183,3 +198,4 @@ def test_hist_number_of_columns(image_array_2bands):
     for n in number_of_columns:
         f, ax = ep.hist(image_array_2bands, cols=n)
         assert [a.numCols for a in ax] == [n] * 2
+    plt.close()
```

Comment:
Should we be closing the plots in the loop here instead? Calling `plt.close()` with no arguments seems to close the current plot, which is probably the last one created: https://matplotlib.org/api/_as_gen/matplotlib.pyplot.close.html

Reply:
oops yes!